snuba/clusters/storage_set_key.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+from typing import Any, Dict, Iterator
+
+HARDCODED_STORAGE_SET_KEYS = {
+    "CDC": "cdc",
+    "DISCOVER": "discover",
+    "EVENTS": "events",
+    "EVENTS_RO": "events_ro",
+    "METRICS": "metrics",
+    "MIGRATIONS": "migrations",
+    "OUTCOMES": "outcomes",
+    "QUERYLOG": "querylog",
+    "SESSIONS": "sessions",
+    "TRANSACTIONS": "transactions",
+    "TRANSACTIONS_RO": "transactions_ro",
+    "TRANSACTIONS_V2": "transactions_v2",
+    "ERRORS_V2": "errors_v2",
+    "ERRORS_V2_RO": "errors_v2_ro",
+    "PROFILES": "profiles",
+    "FUNCTIONS": "functions",
+    "REPLAYS": "replays",
+    "GENERIC_METRICS_SETS": "generic_metrics_sets",
+    "GENERIC_METRICS_DISTRIBUTIONS": "generic_metrics_distributions",
+}
+
+
+REGISTERED_STORAGE_SET_KEYS: Dict[str, str] = {}
```

Comment:
Can we mark this as private? (even if it's just using a leading underscore to convey intent). I am worried about someone accidentally importing this and it being an empty dictionary.

Reply:
good point, I'll change that